### ISSUE TITLE
Is it possible to use this plugin *without* enabling code shrinker but with linking SDK assemblies?

### ISSUE THREAD

**Issue by mikequ-taggysoft** [jamesmontemagno/StoreReviewPlugin#42]
Doc says that if `Linking` is set to SDK or SDK and User Assemblies, we need to create the Proguard file *and* set code shrinker to `Proguard` or `r8`.

But our current project specifically does not want to use code shrinker, even thought it does link SDK assemblies.

We do not want to use the shrinker because it affects other parts of our app and we do not want to mess with it. We tested/compared the size with/without the shrinker and found the difference negligible for our app.

But we do want to use this plugin and keep the SDK linking setting. Is there a way to do it without enabling the shrinker?

Thanks.

**Comment by saamerm** [comment]
Yes you can keep the SDK linking setting without enabling the shrinker and it should still work. @mikequ-taggysoft

**Comment by mikequ-taggysoft** [comment]
> Yes you can keep the SDK linking setting without enabling the shrinker and it should still work. @mikequ-taggysoft

@saamerm That's good to know! In that case, do we still need the proguard file at all?

**Comment by saamerm** [comment]
Yes I believe that is needed, but you can [read more about it here](https://docs.microsoft.com/en-us/xamarin/android/deploy-test/release-prep/proguard?tabs=macos). Is there a reason you dont want to include the pro guard?

**Comment by mikequ-taggysoft** [comment]
@saamerm I don't mind including the proguard file for this particular plugin. I just don't want to use enable the shrinker because I don't want the headache of maintaining exclusions for other classes/packages when the shrinker doesn't benefit our app in any significant way.

**Comment by saamerm** [comment]
Yes then dont shrink your code. You will have no issues, it should work. 
Let me know if you have any questions @mikequ-taggysoft
